pdstemplate/__init__.py

```
@@ -1140,6 +1151,13 @@ def FILE_ZULU(filepath):
             form "yyyy-mm-ddThh:mm:ssZ".
         """
 
+        ##################################################################################
+        logger = get_logger()
+        logger.warning(f'FILE_TIME reflects the cached time: {filepath}', force=True)
```

Ruff (0.14.8)

Logging statement uses f-string

(G004)